### PR TITLE
README: explain asterisk-vanilla and asterisk-debug

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+asterisk (8:16.6.2-1~4) wazo-dev-buster; urgency=medium
```
Comment (fredericlepied) [inline debian/changelog]: it's not ~wazo4 instead of ~4?

Comment (pc-m) [reply debian/changelog]: yeah it should have been `~wazo4`
I'll have to force a rebuild with the good version because ~4 is lower than ~wazo3

**Comment by fblackburn1** [comment]
No need to bump the changelog, jenkins will ignore README.md

**Comment by pc-m** [comment]
> No need to bump the changelog, jenkins will ignore README.md

I know but I wanted to force a rebuild for another branch and I tried to kill 2 birds with one stone by merging this pr with a changelog update but it failed miserably.

**Comment by fblackburn1** [comment]
Kill two birds !!!!!! :scream: 
How dare you !? :feelsgood:

**Pull request merged**